srcpkgs/nginx/template

```diff
@@ -1,16 +1,22 @@
 # Template file for 'nginx'
 pkgname=nginx
-version=1.18.0
-revision=5
+version=1.20.1
+revision=1
+_tests_commit=feb754918372
 build_style=gnu-makefile
 hostmakedepends="openssl-devel pcre-devel $(vopt_if geoip geoip-devel)"
 makedepends="${hostmakedepends}"
+checkdepends="perl perl-IO-Socket-SSL which"
 short_desc="High performance web and reverse proxy server"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-2-Clause"
 homepage="https://nginx.org"
-distfiles="https://nginx.org/download/nginx-${version}.tar.gz"
-checksum=4c373e7ab5bf91d34a4f11a0c9496561061ba5eee6020db272a17a7228d35f99
+distfiles="https://nginx.org/download/nginx-${version}.tar.gz
+ https://hg.nginx.org/nginx-tests/archive/${_tests_commit}.tar.gz"
```

Comment:
With that, files from second archive are left in builddir, because xbps-src removes only `wrksrc`, which is `nginx-${version}` extracted from first archive here.
Add `create_wrksrc=yes` `build_wrksrc="nginx-${version}"` `TEST_NGINX_BINARY="$PWD/objs/nginx"`, so both archives are extracted under `wrksrc` and get removed after build.

Reply:
Hm, then can i ask why `create_wrksrc=yes` is not automated when there is multiple distfiles?

Reply:
Is there any other reason?

Reply:
Sometimes it is required to move secondary archives into main one.
Other times, it's not necessary because one file is archive and others are non-archives marked with skip_extraction and handled separately with vinstall.